### PR TITLE
ci: bump stack versions

### PR DESCRIPTION
**Pull request by v1v**
### What

* Use `7.16.0`
* Remove `7.x` reference
* Pass the STACK_VERSION to the function rather than using the environment variable.

### Issue

Closes https://github.com/elastic/apm-agent-rum-js/issues/1111

### Follow ups

Since `7.x` is not anymore a thing, what do we do with the 8 major since 8.0 branch exists and master points to 8.1 now.?

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -142,11 +142,10 @@ pipeline {
             }
           }
         }
-        stage('Stack 8.0.0-SNAPSHOT SauceLabs') {
+        stage('Stack 8.1.0-SNAPSHOT SauceLabs') {
```
Comment (v1v) [inline Jenkinsfile:145]:
```suggestion
        stage('Stack 8.0.0-SNAPSHOT SauceLabs') {
```

File: Jenkinsfile [hunk 2]
```diff
@@ -167,7 +166,7 @@ pipeline {
             }
           }
           steps {
-            runAllScopes()
+            runAllScopes(stack: "8.1.0-SNAPSHOT")
```
Comment (v1v) [inline Jenkinsfile:169]:
```suggestion
            runAllScopes(stack: "8.0.0-SNAPSHOT")
```

File: Jenkinsfile [hunk 3]
```diff
@@ -115,7 +115,7 @@ pipeline {
                   // The below line is part of the bump release automation
                   // if you change anything please modifies the file
                   // .ci/bump-stack-release-version.sh
-                  values '8.0.0-SNAPSHOT', '7.x', '7.16.0'
+                  values '8.1.0-SNAPSHOT', '8.0.0-SNAPSHOT', '7.16.0'
```
Comment (v1v) [inline Jenkinsfile:118]:
```suggestion
                  values '8.0.0-SNAPSHOT', '7.16.0'
```

File: Jenkinsfile [hunk 4]
```diff
@@ -556,6 +555,7 @@ def prepareRelease(String nodeVersion='node:lts', Closure body){
 }
 
 def runAllScopes(){
+  def stack = args.stack
```
Comment (v1v) [inline Jenkinsfile:558]:
```suggestion
def runAllScopes(Map args = [:]) {
  def stack = args.stack
```

**Review by vigneshshanmugam — APPROVED** [review]
LGTM, test error was unrelated to the PR.

**Pull request merged**